src/sweetyMJun20/BankingSystem.java

```diff
@@ -0,0 +1,52 @@
+package sweetyMJun20;
+
+public class BankingSystem {
+	int creditcnt;
+	int debitcnt;
+	int printcnt;
+	static int usercreditcnt;
+	static int userdebitcnt;
+	static int userprintcnt;
+	
+	void debitAcc(int debtamt){
+        debitcnt++;
+		userdebitcnt++;
+	}
+	void creditAcc(int creditamt){
+		creditcnt++;
+		usercreditcnt++;
```

Comment:
You forgot to perform an operation in debitAcc() and creditAcc() METHODS. You should have a balance instance variable.  You should write the condition to verify the insufficient balance.